### PR TITLE
feat: add panel header name to self registration scheme

### PR DESCRIPTION
**Pull request by drdelambre**
we're currently hand editing all of the "human readable" panel header names. lets just use the panel's type definition to grab the same text as we're putting on the button they click. talking about automating this thing:

Screenshot from 2021-01-29 12-00-01

### CODE REVIEW

File: src/flows/pipes/Youtube/index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export default register => {
+  register({
+    type: 'youtube',
+    family: 'test',
+    priority: -1,
+    featureFlag: 'flow-panel--youtube',
+    button: 'Youtube',
+    component: View,
+    initial: {
+      uri: 'nBHkIWAJitg',
```
Comment (genehynson) [inline src/flows/pipes/Youtube/index.ts:13]: what is this random string?

Comment (genehynson) [reply src/flows/pipes/Youtube/index.ts:13]: oh it's the youtube id of some influx video lol ok

**Review by genehynson — APPROVED** [review]

File: src/flows/pipes/Youtube/index.ts (hunk 1 above)
Comment (drdelambre) [reply src/flows/pipes/Youtube/index.ts:13]: the id that follows a youtube video's url

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 50 | :red_circle:
accessibility | 65 | :yellow_circle:
best-practices | 87 | :green_circle:
seo | 80 | :yellow_circle:
pwa | null | :question:

**Pull request merged**